core/snapshots/snapshotter.go

```diff
@@ -379,6 +382,30 @@ func WithLabels(labels map[string]string) Opt {
 	}
 }
 
+// WithLabels appends labels to a created snapshot
+func WithQuotaSize(size string) Opt {
```

Comment:
Shouldn't this take uint64 as an argument?

Reply:
If it's uint64, obviously it's byteSize, but it's not user-friendly for human readable configurations.

This configuration will eventually be handed by upper layer, such as `Kubernetes`, where readability feels more important, so it was not `uint64` type.

Reply:
Golang API should provide a low-level interface and does not need to cover human user experiences directly.

Reply:
makes sense, has been modified